gatsby-config.ts

```diff
@@ -1,22 +1,24 @@
-require('dotenv').config({ path: 'vtex.env' })
+import { join, resolve } from 'path'
 
-const { join, resolve } = require('path')
+import dotenv from 'dotenv'
+import type { GatsbyConfig } from 'gatsby'
 
-const { getSchema, getContextFactory } = require('./src/server')
-const config = require('./store.config')
+import config from './store.config'
+
+dotenv.config({ path: 'vtex.env' })
 
 const {
   NODE_ENV,
   URL = config.storeUrl,
   DEPLOY_PRIME_URL = URL,
   CONTEXT: ENV = NODE_ENV,
```

Comment:
While we're here, do you think this can be removed as well? It's not being used within this file.

![CleanShot 2022-04-04 at 15 43 54](https://user-images.githubusercontent.com/381395/161610684-e26575c9-a9e2-4538-92a7-7eebf46f1d6e.png)

```suggestion
```

Reply:
we use for `ENV`, that is used. for `isProduction` etc..

Reply:
I know about `ENV`, but I don't see `CONTEXT` being **directly** used in the file so I thought it wasn't being used.

Reply:
Oh, my bad! I didn't realize `CONTEXT` here was actually being renamed to `ENV`! 🤦‍♂️